README.md

```diff
@@ -1,5 +1,25 @@
-Android User Interface Samples Repository
-=========================================
+# Android user interface samples
 
-This repository contains a set of individual Android Studio projects to help you get
-started writing/understanding Android user interface features.
+A collection of Android Studio projects to help you start building Android user interface features.
```

Comment:
Probably can remove "Studio"? Calling them an Android Studio project sounds a bit strange, even if that's the tool to edit them with.

```suggestion
A collection of Android projects to help you start building Android user interface features.
```